.github/workflows/go.yml

```diff
@@ -3,10 +3,12 @@ on:
   pull_request:
     branches:
     - master
+    - windows
```

Comment:
@ruicao93 I think you can remove the "windows" branch occurrences from the github CI files

Reply:
Thanks @antoninbas .

I create a PR to remove windows from github CI. Could you help approve it?
https://github.com/vmware-tanzu/antrea/pull/741

I will then update this PR to include #741.